src/Providers/StyleProvider.tsx

```diff
@@ -11,6 +11,9 @@ const appTheme = createMuiTheme({
     palette: {
         primary: {
             main: '#fafafa'
+        },
+        secondary: {
+            main: '#282828'
```

Comment:
You can discard this changes, so there won't be merge conflict with #3 
btw primary and secondary are accent colors, not background ;d